lms/djangoapps/commerce/views.py

```diff
@@ -79,6 +79,23 @@ def post(self, request, *args, **kwargs):  # pylint: disable=unused-argument
         if not valid:
             return DetailResponse(error, status=HTTP_406_NOT_ACCEPTABLE)
 
+        # Ensure that the course has an honor mode with SKU
+        honor_mode = CourseMode.mode_for_course(course_key, CourseMode.HONOR)
```

Comment:
I'm not sure what's going on here. However, looking at the commit history for this PR (https://github.com/edx/edx-platform/pull/7426/commits), it appears my commit was applied twice, once for the merge and a second time for...I don't know what. Removing this second, mystery, commit (d6046d786d3d72ee298ce6e3a682aa7a946196af) *should* resolve this problem.